client/extensions/woocommerce/app/products/product-form.js

```diff
@@ -2,15 +2,12 @@
  * External dependencies
  */
 import React, { Component, PropTypes } from 'react';
-import i18n from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-import FoldableCard from 'components/foldable-card';
 import ProductFormDetailsCard from './product-form-details-card';
-import ProductVariationTypesForm from './product-variation-types-form';
-import FormToggle from 'components/forms/form-toggle';
+import ProductFormVariationsCard from './product-form-variations-card';
 
 export default class ProductForm extends Component {
```

Comment:
Now that this component is stateless, it can be made functional. I don't know if you folks have a hard rule about that.